src/bitgraphs.jl

```diff
@@ -0,0 +1,117 @@
+mutable struct BitGraph{INT}
+    N::Int # size of the graph
+    bitgraph::Vector{INT} # store the adjacency matrix as vector of BitStr
+    fadjlist::Vector{Vector{Int}} # store the adjacency list, informate of sparse graph
+    mask::INT # mask for the graph
```

Comment:
What about changing the struct name to `MaskedBitGraph`?

Reply:
changed